derive/src/lib.rs

```diff
@@ -84,7 +96,7 @@ pub fn contract(input: TokenStream) -> TokenStream {
 
     let span = args.span();
     expand(args.into_inner())
-        .unwrap_or_else(|e| SynError::new(span, e.to_string()).to_compile_error())
+        .unwrap_or_else(|e| SynError::new(span, format!("{:?}", e)).to_compile_error())
```

Comment:
Is this change intended?

Reply:
Yes, it makes compile errors easier to debug. I've opened a separate PR #239 as its technically unrelated.